set.mm

```diff
@@ -528259,134 +529263,326 @@ C Fn ( ( S i^i dom F ) u. { z } ) ) $=
+    $=
+      ( wa cslt wral csur wss wcel cv wbr w3a cvv con0 cbday cun cima wrex elex
+      cfv anim2i id 3anim123i cuni csuc cdm cdif c2o csn cxp c1o eqid noetalem1
+      wo wceq breq2 ralbidv breq1 fveq2 sseq1d 3anbi123d rspcev jaoi syl sylan
+      ) EUAUBZEKUCZRZFUAUBZFLUCZRZMUDZNUDZSUENFTMETZUFVTEUGUCZRZWCFUGUCZRZWHUFZ
+      JUHUCUIEFUJUKJUBRZWFOUDZSUEZMETZWOWGSUEZNFTZWOUIUNZJUBZUFZOUAULZWBWJWEWLW
+      HWHWAWIVTEKUMUOWDWKWCFLUMUOWHUPUQWMWNRGUAUCWFGSUEZMETZGWGSUEZNFTZGUIUNZJU
+      BZUFZRZHUAUCWFHSUEZMETZHWGSUEZNFTZHUIUNZJUBZUFZRZVHXCABCDEFGHIJHUIEUKURUS
+      HUTVAVBVCVDUJZGUIFUKURUSGUTVAVEVCVDUJZMNPQYAVFXTVFVGXKXCXSXBXJOGUAWOGVIZW
+      QXEWSXGXAXIYBWPXDMEWOGWFSVJVKYBWRXFNFWOGWGSVLVKYBWTXHJWOGUIVMVNVOVPXBXROH
+      UAWOHVIZWQXMWSXOXAXQYCWPXLMEWOHWFSVJVKYCWRXNNFWOHWGSVLVKYCWTXPJWOHUIVMVNV
+      OVPVQVRVS $.
   $}
 
+
   ${
-    $d A a b c d e f x y z $.  $d B x y z $.
+    $d A a $.  $d a b $.  $d A b $.  $d a c $.  $d A c $.  $d a d $.  $d A d $.
+    $d a e $.  $d A e $.  $d a f $.  $d A f $.  $d a g $.  $d A g $.  $d a h $.
+    $d A h $.  $d a j $.  $d A j $.  $d a k $.  $d A k $.  $d a x $.  $d A x $.
+    $d a y $.  $d A y $.  $d A z $.  $d B a $.  $d B b $.  $d b c $.  $d B c $.
+    $d b d $.  $d B d $.  $d b e $.  $d B e $.  $d b f $.  $d B f $.  $d b g $.
+    $d B g $.  $d B h $.  $d b j $.  $d B j $.  $d b k $.  $d B k $.  $d b x $.
+    $d B x $.  $d b y $.  $d B y $.  $d B z $.  $d c d $.  $d c e $.  $d c f $.
+    $d c g $.  $d c h $.  $d c j $.  $d c k $.  $d c x $.  $d c y $.  $d d e $.
+    $d d f $.  $d d g $.  $d d h $.  $d d j $.  $d d k $.  $d d x $.  $d e f $.
+    $d e g $.  $d e h $.  $d e j $.  $d e k $.  $d e x $.  $d e y $.  $d f g $.
+    $d f h $.  $d f j $.  $d f x $.  $d f y $.  $d f z $.  $d g h $.  $d g j $.
+    $d g k $.  $d g x $.  $d g y $.  $d g z $.  $d h j $.  $d h k $.  $d h x $.
+    $d h y $.  $d h z $.  $d j k $.  $d j x $.  $d j y $.  $d j z $.  $d k x $.
+    $d k y $.  $d k z $.  $d O b $.  $d O d $.  $d O z $.  $d x y $.  $d x z $.
+    $d y z $.
     $( The full-eta axiom for the surreal numbers.  This is the single most
        important property of the surreals.  It says that, given two sets of
        surreals such that one comes completely before the other, there is a
        surreal lying strictly between the two.  Furthermore, there is an upper
        bound on the birthday of that surreal.  Alling's axiom FE. (Contributed
-       by Scott Fenton, 7-Dec-2021.) $)
-    noeta $p |- ( ( ( A C_ No /\ A e. V ) /\ ( B C_ No /\ B e. W ) /\
-            A. x e. A A. y e. B x <s y ) ->
+       by Scott Fenton, 9-Aug-2024.) $)
+    noeta $p |- ( ( ( ( A C_ No /\ A e. V ) /\ ( B C_ No /\ B e. W ) /\
+            A. x e. A A. y e. B x <s y ) /\
+            ( O e. On /\ ( bday " ( A u. B ) ) C_ O ) ) ->
           E. z e. No ( A. x e. A x <s z /\ A. y e. B z <s y /\
-                       ( bday ` z ) C_ suc U. ( bday " ( A u. B ) ) ) ) $=
```

Comment:
Since you added an antecedent in the formalization, you should also add that condition in the comment; something like
> ... and the birthdays of members of A and B are bounded, there is a surreal lying strictly between the two sets, whose birthday is bounded by the successor of the union of the set of birthdays of the surreals in these sets.

I'm not familiar enough with surreals, so you'll find the correct formulation.